app/views/admin/settings/discovery/show.html.haml

```diff
@@ -42,6 +42,13 @@
               label: t('admin.settings.default_noindex.title'),
               wrapper: :with_label
 
+  .fields-group
```

Comment:
I would add a `%h4` header titled "Privacy" above the `noindex` setting.

Reply:
![Screenshot 2025-05-19 at 13 42 46](https://github.com/user-attachments/assets/570feb77-94ae-4663-92c2-8737a6bff8c3)

@Gargron here's the full page with this new header. i think this looks too busy? what i'd propose:

- we combine `Publish statistics` and `Publish discovered servers` into one header, perhaps `Publish server information`
- move `Allow unauthenticated access to public timelines` under the new `Privacy` header
- get rid of the `Public timelines` header

Reply:
I agree it doesn't make sense for the two bottom items to each have a separate category, I'd maybe just leave the "Public statistics" header for both. Unauthenticated access to public timelines is not as related to user privacy so I'd maybe leave it as-is for now and come back to it when we approach some other logged-out-experience reworks.